### PR TITLE
Update Android base package name

### PR DESCRIPTION
**Pull request by merulaxr**
Updates apk package name to reflect our correct domain as we publish to more platforms.

Override specifically for Oculus platform as we're already published there so we're locked in.

### CODE REVIEW

File: Assets/Editor/BuildTiltBrush.cs [hunk 1]
```diff
@@ -961,6 +961,10 @@ public TempSetAppNames(bool isAndroid, string Description)
             m_company = PlayerSettings.companyName;
             string new_name = App.kAppDisplayName;
             string new_identifier = GuiBuildAndroidApplicationIdentifier;
+#if OCULUS_SUPPORTED
+            //Can't change Quest identifier
+            new_identifier = "com.Icosa.OpenBrush";
+#endif
```
Comment (mikeage) [inline Assets/Editor/BuildTiltBrush.cs:967]: Does this make it into the manifest? If so, can we remove that package=XXX completely? If not, is this a problem?

Comment (merulaxr) [reply Assets/Editor/BuildTiltBrush.cs:967]: I'll check the CI when it finishes, but yes this should override the manifest template, per my mistake in #280 that was fixed with #281

Comment (merulaxr) [reply Assets/Editor/BuildTiltBrush.cs:967]: Yes looks to have worked.

Comment (mikeage) [reply Assets/Editor/BuildTiltBrush.cs:967]: awesome! Should we replace the one in the manifest with something like `com.to.be.replaced.at.build.time`?

Comment (merulaxr) [reply Assets/Editor/BuildTiltBrush.cs:967]: We could probably remove the line entirely, I could give that a shot!

**Pull request merged**